changelogs/fragments/ansible_test_rmv_macos_132.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - ansible-test - Removed test remotes: macos/13.2
```

Comment:
This is now a list with a single-element dictionary.
```suggestion
  - "ansible-test - Removed test remotes: macos/13.2"
```

Reply:
Ah, right,thanks!